pkg/reconciler/inmemorychannel/dispatcher/controller.go

```diff
@@ -45,9 +47,12 @@ import (
 	tracingconfig "knative.dev/pkg/tracing/config"
 
 	"knative.dev/eventing/pkg/apis/eventing"
+	messagingv1 "knative.dev/eventing/pkg/apis/messaging/v1"
 	"knative.dev/eventing/pkg/channel"
 	eventingclient "knative.dev/eventing/pkg/client/injection/client"
 	inmemorychannelinformer "knative.dev/eventing/pkg/client/injection/informers/messaging/v1/inmemorychannel"
+	inmemorychannelreconciler "knative.dev/eventing/pkg/client/injection/reconciler/messaging/v1/inmemorychannel"
+	v1 "knative.dev/eventing/pkg/client/listers/messaging/v1"
```

Comment:
nit: `v1` sounds vague to me. `messaginglisters` or `messaginglistersv1` would be more aligned with the aliases used above, and clearly indicate what import we are talking about.